Makefile

```diff
@@ -40,7 +40,7 @@ ifeq ($(OSFLAG),$(LINUX))
 	curl -s https://raw.githubusercontent.com/k3d-io/k3d/main/install.sh | bash
 ifneq ($(CI),true)
 	# install nix
-	sh <(curl -L https://nixos-nix-install-tests.cachix.org/serve/vij683ly7sl95nnhb67bdjjfabclr85m/install) --daemon --tarball-url-prefix https://nixos-nix-install-tests.cachix.org/serve --nix-extra-conf-file ./nix.conf
+	# sh <(curl -L https://nixos-nix-install-tests.cachix.org/serve/vij683ly7sl95nnhb67bdjjfabclr85m/install) --daemon --tarball-url-prefix https://nixos-nix-install-tests.cachix.org/serve --nix-extra-conf-file ./nix.conf
```

Comment:
@archseer do you know why this is failing? I commented it out since it was blocking things

Reply:
Yeah I'm not sure why this is here, maybe it was intended to be used on the CI

Reply:
@tateexon originally added this in AFAIK

Reply:
i'll leave this commented out for now as it doesn't seem to be used - it can be revisited later on if needed